### ISSUE TITLE
Add Work Type on Challenge listing grid

### ISSUE THREAD

**Issue by drasticdpk** [topcoder-platform/challenge-engine-ui#819]
**Reference**
 
![image](https://user-images.githubusercontent.com/25297859/93438158-a4dae280-f8ec-11ea-89e8-88d405e497fd.png)

**Comment by vikasrohit** [comment]
Thanks for the suggestion @drasticdpk. Right now, the background color of the work format e.g. `First2Finish` in your example, depicts the work type. They match with colors of the icons for the work type options e.g. in your example it is a Development First2Finish.

**Comment by drasticdpk** [comment]
Ok great. It will take take some time to familiar with color. 
Shall I close it ? cc  @vikasrohit

**Comment by vikasrohit** [comment]
Yes, for now. I agree it might need better UX for getting that logic conveyed to the end user and I hope that the new designs take care of that.
